source/docs/software/driverstation/driver-station.rst

```diff
@@ -19,10 +19,12 @@ Setting Up the Driver Station
 
 .. image:: images/driver-station/ds-setup.png
 
-The DS must be set to your team number in order to connect to your robot. In order to do this click the Setup tab then enter your team number in the team number box. Press return or click outside the box for the setting to take effect.
+The DS should be set to your team number in order to connect to your robot. In order to do this click the Setup tab then enter your team number in the team number box. Press return or click outside the box for the setting to take effect.
 
 PCs will typically have the correct network settings for the DS to connect to the robot already, but if not, make sure your Network adapter is set to DHCP.
 
+.. note:: An IP address can also be entered in the :guilabel:`Team Number` box for use in rare situations.  This is not recommended for normal use!
```

Comment:
Maybe instead of just saying "rare situations" here we elaborate a little more on what situations? They may not end up being all that rare. Situations I can think of: non-RoboRIO targets, troubleshooting, possibly simulation.

Reply:
If there is no team number entered in the box (either when XXYY or when cleared) what will the dropdown show?  

I would in general argue that showing all the robots is a good idea for the single click and might be more intuitive.  What is the downside to always showing all of them?  Do we expect the robot to be on networks with that many roboRIOs?

Reply:
How about
```
.. note:: An IP address can also be entered in the :guilabel:`Team Number` box as an alternative. This is not recommended for competition use!
```